_i18n/en/resources/user-guides/mine-to-pool.md

```diff
@@ -1,111 +1,197 @@
-{% assign version = '1.1.1' | split: '.' %}
+{% assign version = '1.2.0' | split: '.' %}
 {% include disclaimer.html translated="true" version=page.version %}
-## Selecting a pool
+## Wallet
 
-There are many pools to choose from, a list is available at
-[moneropools.com](https://moneropools.com). Mining on a larger pool could mean
-more frequent payouts, but mining on a smaller pool helps to keep the network
-decentralized.
+Before starting, you already need to have a wallet configured and
+working. The pool needs to know your wallet address to be able to send
+payments there. See the [Accepting Minero guide]({{ site.baseurl
```

Comment:
`Minero`. Same for all languages

Reply:
Fixed, also got rid of the TODO comment.